aws-go-fargate/README.md

```diff
@@ -31,16 +31,10 @@ Next, to deploy the application and its infrastructure, follow these steps:
     $ pulumi config set aws:region us-east-1 # any valid AWS region will work
     ```
 
-3. Restore your Go dependencies. This example currently uses [Dep](https://github.com/golang/dep) to do so:
+1. Restore your Go dependencies. This example currently uses [go modules](https://github.com/golang/go/wiki/Modules) to do so:
 
     ```bash
-    $ dep ensure
-    ```
-
-4. Compile your Go application:
-
-    ```bash
-    $ go build .
+    $ go mod download
```

Comment:
Users don't actually have to do this do they?  It gets automatically done as needed.  I think we can leave out of instructions.

Reply:
Good point. go run and go build both acquire dependencies automatically. Removed.